packages/frontend2/src/server/features/bridges/get-bridge-risk-entries.ts

```diff
@@ -0,0 +1,57 @@
+import { type Bridge, type Layer2, bridges } from '@l2beat/config'
+import { notUndefined } from '@l2beat/shared-pure'
+import { getImplementationChangeReport } from '../implementation-change-report/get-implementation-change-report'
+import { getLatestTvlUsd } from '../scaling/tvl/utils/get-latest-tvl-usd'
+import { orderByTvl } from '../scaling/tvl/utils/order-by-tvl'
+import { isAnySectionUnderReview } from '../scaling/utils/is-any-section-under-review'
+import { getProjectsVerificationStatuses } from '../verification-status/get-projects-verification-statuses'
+import { getDestination } from './get-destination'
+import { type BridgesRiskEntry } from './types'
+
+export async function getBridgeRiskEntries(): Promise<BridgesRiskEntry[]> {
+  const [tvl, projectsVerificationStatuses, implementationChangeReport] =
+    await Promise.all([
+      getLatestTvlUsd(),
+      getProjectsVerificationStatuses(),
+      getImplementationChangeReport(),
+    ])
```

Comment:
I guess this should be moved to page for this function to be easily testable.

Reply:
I guess we need to talk about it then as I was the person that opted for moving it inside files 😅